hack/db-resources.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+# WARNING: this is intended to be applied in a single node
+#   Kubernetes environment and just for testing purposes
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: databases
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    app: postgresql
+  name: postgresql
+  namespace: databases
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  labels:
+    type: local
+    app: postgresql
+  name: postgresql
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: /data/postgresql
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: postgresql
+  name: postgresql
+  namespace: databases
+spec:
+  selector:
+    matchLabels:
+      app: postgresql
+  template:
+    metadata:
+      labels:
+        app: postgresql
+    spec:
+      containers:
+        - name: postgresql
+          image: postgres:16
+          ports:
+            - containerPort: 5432
+          volumeMounts:
+            - name: postgresql
+              mountPath: /var/lib/postgresql/data
+          env:
+            - name: POSTGRES_DB
+              value: root
+            - name: POSTGRES_USER
+              value: root
+            - name: POSTGRES_PASSWORD
+              value: password  # notsecret
```

Comment:
Why are these not the same values that are in the `values.yaml` file?

Reply:
These define the superadmin user for postgresql, I don't think that running KubeArchive with the superadmin user is realistic, so I used these for the production-like deployment (ci).

Reply:
I don't understand.  Why not just put in the real development values?

Reply:
IINM, what @rh-hemartin tried to do in the original `hack/install-database.sh` script (where this piece comes from) was to replicate the user situation as much as possible.

Before installing KubeArchive, a user will probably have a Kubernetes database deployed somewhere (outside of kubearchive namespace, maybe even outside of the cluster) and with at least a user with admin privileges.

We ask the user to run the DDL and set things up so there is a kubearchive database user with which interact.

So, taking this under consideration, for testing purposes:
* We deploy a db in a namespace different from `kubearchive` (but inside the cluster for convenience) 
* The DB has one user database with admin privileges different from `kubearchive`, `root` in this case.
* This admin is the one that is going to perform the operations of creating a user

That's why the deployment of the datbase doesn't have the "development" values, because they are inserted after the database is deployed in the `install.sh` script.

Reply:
Well, this file appears to be gone now.  I'll drop this for now.